test/main.cc

```diff
@@ -415,15 +415,22 @@ int main(int argc, char* argv[])
         }
 
         // Run the step
-        hpwh.runOneStep(allSchedules[0][i],           // Inlet water temperature (C)
-                        GAL_TO_L(allSchedules[1][i]), // Flow in gallons
-                        airTemp2,                     // Ambient Temp (C)
-                        allSchedules[3][i],           // External Temp (C)
-                        drStatus, // DDR Status (now an enum. Fixed for now as allow)
-                        1. * GAL_TO_L(allSchedules[1][i]),
-                        allSchedules[0][i],
-                        vectptr);
-
+        try
+        {
+            hpwh.runOneStep(allSchedules[0][i],           // Inlet water temperature (C)
+                            GAL_TO_L(allSchedules[1][i]), // Flow in gallons
+                            airTemp2,                     // Ambient Temp (C)
+                            allSchedules[3][i],           // External Temp (C)
+                            drStatus, // DDR Status (now an enum. Fixed for now as allow)
+                            1. * GAL_TO_L(allSchedules[1][i]),
+                            allSchedules[0][i],
+                            vectptr);
+        }
+        catch (...)
+        {
+            cout << "Error in hpwh.runOneStep.\n";
+            exit(1);
+        }
```

Comment:
Is this necessary? Won't an uncaught exception effectively terminate the program with a non-zero return code? It probably doesn't really matter one way or the other, but it is more code than was there before.